### ISSUE TITLE
metatrader tidak bisa connect ke binary server

### ISSUE THREAD

**Issue by carlordum** [Fruitfella/ROBOT-binary#308]
metatrader tidak bisa connect ke binary server bbrp hari ini, apa ada yang menggunakan metatrader 5 di sini?

**Comment by valentino242** [comment]
disaya connect bro.  Bikin new login aja . Biasanya beres.

**Comment by carlordum** [comment]
uda bikin new login bro, sampai 4 username baru, ga bs" jg,
aneh bener ini, saya cb metatrader 5 sama 4 dr broker lain bisa, saya uda coba pakai virtual box dengan windows yg laen, ga bs tembus jg, uda nanya ama admin binary nya, tp ga ada solusi yang tepat, 
padahal da enak pake metatrader nya.

**Comment by Zul79** [comment]
Aku juga nggak bisa log in ke versi desktop platform, tapi kalo versi web platform bisa, tapi aku butuh banget histrorical data Index volatility 100 untuk back testing dan nggak ada vasilitas downloadnya di versi web platform...

**Comment by Zul79** [comment]
Yeah, aku sudah bisa masuk ke server mt5 binary.com versi desktop platform... entah kebetulan atau tidak, caranya adalah jalankan aplikasi metatrader 5 binary.com, lalu klik file-->open an account-->muncul open an account dialog box-->pindahkan kursor ke dalam kotak "add new broker", lalu ketikkan Binary.com, dan klik tombol "find your broker" di samping kotak pencarian itu, lalu di bawahnya akan muncul Binary limited dengan icon binary.com-->pilih icon itu, dan klik tombol "next" di bawahnya-->Lalu kotak dialog open an account akan muncul-->klik tombol option, lalu isikan id dan password mt5 binary.com kamu, dan ten ten aplikasi MT5 mu mungkin segera akan terkoneksi....
